### DISCUSSION TITLE
How to get a scatter chart.

### DISCUSSION THREAD

**Discussion by wangqi1221** [patrykandpatrick/vico#658] [Questions]
Hi, I want to get a scatter chart, but I don't know how to do? Could you share a demo? Thanks!

**Comment by Gowsky** [comment]
Hi! Dedicated scatter charts are a planned addition. For the time being, you may use a line chart with the line width set to 0. Then you'll need to specify a `point` and its size. 

I can show you the example code, but please let me know what Vico version you're using and whether you're using Jetpack Compose or the view system.

**Reply by baum-sparton** [reply]
Any status update on this?  Did an example get posted?  I'd be interested in one for the latest release 2.0.0-beta.1.
A couple things I'm not sure would be supported by the line chart are:
1) Different colors for different points (perhaps this would be possible by plotting a different line for each desired color).
2) With a scatter plot there could potentially be multiple values for the same X.

**Reply by Gowsky** [reply]
>Did an example get posted?

No, as the Vico version and the UI framework were never specified.

>Different colors for different points (perhaps this would be possible by plotting a different line for each desired color).

Yes, you can use multiple series, so they can have various colors.

>With a scatter plot there could potentially be multiple values for the same X.

A single series can't have multiple values for a single _x_ value. However, _x_ values can be repeated between series.
